src/main/java/com/dev/cinema/model/User.java

```diff
@@ -0,0 +1,67 @@
+package com.dev.cinema.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue
+    private Long id;
+    @Column(unique = true)
+    private String email;
+    private String password;
+    private byte[] salt;
+
+    public User() {
```

Comment:
why would you declare this constructor?

Reply:
Because if we have not empty constructor, we have to declare the empty one manually :)